apps/loan/src/utils/helpers.ts

```diff
@@ -93,23 +93,6 @@ export function fulfilledValue<T>(result: PromiseSettledResult<T>) {
   }
 }
 
-export function logTxReceipt<T>(title: string, chainId: ChainId, txReceipt: T & { blockHash?: string; id?: string }) {
```

Comment:
Also unrelated to the PR's goals, but then again an unused function, can be removed.